lib/structurely.rb

```diff
@@ -1,20 +1,26 @@
-require "structurely/version"
 require "api_struct"
 require "dry-configurable"
+require "structurely/version"
+require "structurely/configuration"
 
-require "structurely/settings"
-require "structurely/api_struct_settings"
+module Structurely
+  class << self
+    attr_accessor :configuration
+  end
+
+  def self.configure
+    self.configuration ||= Configuration.new
+    yield(configuration)
+  end
+end
 
+require "structurely/clients/base_client"
```

Comment:
Does Zeitwerk obsolete these? I guess I need to look at the gemspec and see if its listed.